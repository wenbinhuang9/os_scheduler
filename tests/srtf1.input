1 0 6
2 0 8
3 0 7
4 0 3