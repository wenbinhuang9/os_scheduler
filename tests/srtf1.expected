1 9 3 9
2 24 16 24
3 16 9 16
4 3 0 3
7 13